boards.txt

```diff
@@ -295,6 +296,11 @@ esp32s2.menu.DebugLevel.debug.build.code_debug=4
 esp32s2.menu.DebugLevel.verbose=Verbose
 esp32s2.menu.DebugLevel.verbose.build.code_debug=5
 
+esp32s2.menu.USBStack.idf=ESP-IDF
+esp32s2.menu.USBStack.idf.build.usbstack_flags="-I{compiler.sdk.path}/include/arduino_tinyusb/tinyusb/src" "-I{compiler.sdk.path}/include/arduino_tinyusb/include"
```

Comment:
include path to correct tusb_config.h for compiling the correct stack provided by IDF or library version. When libarary version is selected, the actual function implementation in the library is favored by linker by default instead of one in `arduino_tinyusb.a`